dev-docs/bidders/mediago.md

```diff
@@ -26,7 +29,8 @@ The MediaGo Bidding adapter requires setup before beginning. Please contact us a
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description           | Example   | Type      |
 |---------------|----------|-----------------------|-----------|-----------|
-| `token`      | required | publisher token        | `'1e100887dd614b7f69fdd1360437'`    | `string` |
+| `token`      | required | publisher token, This parameter expects all imps to be the same        | `'1e100887dd614b7f69fdd1360437'`    | `string` |
+| `region`      | recommend | Server region for PBS request: US for US Region, EU for EU Region, APAC for APAC Region, default is US. This parameter expects all imps to be the same.         | `'US'`    | `string` |
```

Comment:
Please mark this parameter as being available for PBS only. Apparently the "test" and "bidfloor" parameters are PBJS only. (though we'd prefer you got rid of the bidfloor param and just read the floor in the standard way.

Reply:
@bretg Thanks for your advice. I have changed it.